src/pkg/backup/details/groups.go

```diff
@@ -37,25 +37,32 @@ func NewGroupsLocationIDer(
 
 // GroupsInfo describes a groups item
 type GroupsInfo struct {
+	ItemType ItemType  `json:"itemType,omitempty"`
+	Modified time.Time `json:"modified,omitempty"`
+
+	// Channels Specific
+	Message   ChannelMessageInfo `json:"message"`
```

Comment:
Should we be worried about not being able read older details file? I guess it should be fine as we still mention that "groups" is still experimental, but wanted to bring it up in case this was something you had overlooked.

Reply:
I'm leaning heavily on "not yet released" here.  We've communicated to the CLI community that things may break or change before full release.  And the SDK community will receive similar communication.